doc/news/changes/incompatibilities/20171117Bangerth

```diff
@@ -0,0 +1,15 @@
+Changed: The many KellyErrorEstimator::estimate() overloads all took
+function arguments (or FunctionMap arguments) indicating the Neumann
```

Comment:
Can you please explain why have you described this as 

> function arguments (or FunctionMap arguments)

? As far as I can ell, all of the changed arguments are of type `FunctionMap`.  Maybe this relates to my next question below...